src/platform/backends/qemu/dnsmasq_server.cpp

```diff
@@ -52,6 +52,13 @@ mp::DNSMasqServer::DNSMasqServer(const Path& data_dir, const QString& bridge_nam
     conf_file.open();
     conf_file.close();
 
+    QFile fd_Hosts(QDir(data_dir).filePath("dnsmasq.hosts"));
```

Comment:
Could you change the variable name to `fd_hosts` instead to match the naming scheme of other variables?